crates/uv-python/src/macos_dylib.rs

```diff
@@ -0,0 +1,47 @@
+use std::path::PathBuf;
+
+use uv_fs::Simplified as _;
+
+pub fn patch_dylib_install_name(dylib: PathBuf) -> Result<(), Error> {
+    // Check that `install_name_tool` is available before
+    // trying to run it.
+    if std::process::Command::new("install_name_tool")
+        .arg("--version")
+        .output()
+        .is_err()
+    {
+        return Ok(());
+    }
+
+    let output = std::process::Command::new("install_name_tool")
+        .arg("-id")
+        .arg(&dylib)
+        .arg(&dylib)
+        .output()?;
```

Comment:
Can we do this in a single invocation so we don't pay the overhead of invoking the tool twice? I think you should be able to match the error to determine if it failed because the command was missing.

Reply:
Yes we can! Done in [95d3adf](https://github.com/astral-sh/uv/pull/10629/commits/95d3adf851f75b53b5b52f35aee27e5e685dbe8a)